api/models/task.py

```diff
@@ -9,4 +18,7 @@ class Task(db.Model):
         UUIDType(binary=True), ForeignKey("current_session.user_id"), nullable=False
     )
     title = Column(String, nullable=False)
+    description = Column(String, nullable=False)
+    completed = Column(Boolean, nullable=False)
```

Comment:
`completed` shouldn't be a boolean but rather a (nullable) slot identifier which specifies which slot the task has been completed in (if not null)

this allows us to query tasks complete in a given slot. On new session creation we will simply delete all tasks that have null there, so this should be up to date

Reply:
Yes, it should be a foreign key